src/main/res/values/strings.xml

```diff
@@ -388,13 +388,13 @@
 
     <string name="notification_comment">%s replied to you</string>
     <string name="notification_message">%s sent you a message</string>
-    <string name="notification_message_multiple">New messages on reddit</string>
+    <string name="notification_message_multiple">New messages on Reddit</string>
     <string name="notification_message_action">Touch to view</string>
 
     <!-- General -->
 
-    <string name="general_true">True</string>
-    <string name="general_false">False</string>
+    <string name="general_true">Yes</string>
+    <string name="general_false">No</string>
```

Comment:
Arguable whether this looks better in the UI, but this isn't the place to change it (the string "general_true" should map to "true", not "yes")